.buildkite/commands/lint.sh

```diff
@@ -12,8 +12,13 @@ bundle exec fastlane check_declared_locales_consistency app:"$1"
 echo "--- :microscope: Linting"
 
 if [ "$1" = "wordpress" ]; then
+  set +e
 	./gradlew lintWordpressVanillaRelease
-	exit 0
+	lint_exit_code=$?
+	set -e
+
+	upload_sarif_to_github "WordPress/build/reports/lint-results-wordpressVanillaRelease.sarif"
```

Comment:
We can't send 2 report files, so I decided to stick to WordPress one for no specific reason over Jetpack.

Reply:
If I were to chose WordPress or Jetpack @wzieba , I would personally chose Jetpack instead. I would do so because the WordPress app is a subset of the Jetpack app and doesn't contain every feature. Having said that, I understand  that the codebases is almost fully shared between the two and that those Jetpack features not available to WordPress is behind a feature flag or an if/else statement of some kind, located on the `main` sourceset, which means, it applies to both apps. Still, I think, even conceptually, if I had the choice, I would chose Jetpack. 😊

Reply:
Gotcha - thanks! I've updated in cc99dd3

Reply:
Awesome thanks! 🚀